src/Microsoft.Azure.NotificationHubs.DotNetCore.Tests/ManagementApiE2ETests.cs

```diff
@@ -31,9 +48,16 @@ public ManagementApiE2ETests()
             _notificationHubConnectionString = Environment.GetEnvironmentVariable(NotificationHubConnectionString.ToUpper()) ?? configuration[NotificationHubConnectionString];
             _notificationHubName = Environment.GetEnvironmentVariable(NotificationHubName.ToUpper()) ?? configuration[NotificationHubName];
 
+            _storageAccount = Environment.GetEnvironmentVariable(StorageAccount.ToUpper()) ?? configuration[StorageAccount];
```

Comment:
Just for culture sake, we should probably use `[String.ToUpperInvariant()`](https://docs.microsoft.com/en-us/dotnet/api/system.string.toupperinvariant?view=netframework-4.8)